README.md

```diff
@@ -32,3 +32,78 @@ To only generate the policies in the `helm` folder structure:
 ```bash
 make generate
 ```
+
+### Adding tests
+
+This repository uses the [app-build-suite](https://github.com/giantswarm/app-build-suite/) and the related testing setup.
+We have tried to make the test setup as simple as possible but some python knowledge is required.
+
+The tests use [python fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) extensively to set up any resources we need in our tests.
+
+All fixtures can be found in [ensure.py](https://github.com/giantswarm/kyverno-policies/blob/main/helm/tests/ensure.py).
+Each fixtures should be structured in a similar way:
+```python
+@pytest.fixture
+def myresource(kubernetes_cluster):
+    # Yaml of whatever resource you want to create.
+    c = dedent(f"""
+        apiVersion: cluster.x-k8s.io/v1alpha3
+        kind: MyResource
+        metadata:
+          name: {cluster_name}
+        spec:
+          some:
+            spec: ""
+    """)
+
+    # Creating the resource for our test.
+    kubernetes_cluster.kubectl("apply", input=c, output=None)
+    LOGGER.info(f"MyResource {cluster_name} applied")
```

Comment:
I may be being blind but where is `cluster_name` defined?

Reply:
Good point - it's a global variable in the `ensure.py` file currently. Will add some explanation maybe:
https://github.com/giantswarm/kyverno-policies/blob/9ef729d05d86f5b250f7a6f035862e9f61255f9a/helm/tests/ensure.py#L14

Reply:
Right, yes I see, it makes sense once you open the file 😅 

Yeah but maybe just a word that when you define your fixture function there are some globals you can use.